src/liboslexec/builtindecl.h

```diff
@@ -376,11 +376,11 @@ DECL (osl_trace_set_traceset, "xXs")
 DECL (osl_trace, "iXXXXXXXX")
 
 #ifdef OSL_LLVM_NO_BITCODE
-DECL (osl_calculatenormal, "xXXX")
+DECL (osl_calculatenormal, "xXXi")
 DECL (osl_area, "fX")
 DECL (osl_filterwidth_fdf, "fX")
 DECL (osl_filterwidth_vdv, "xXX")
-DECL (osl_raytype_bit, "iXi")
+DECL (osl_raytype_bit_from_name, "iXi")
```

Comment:
I don't see `osl_raytype_bit_from_name` referenced anywhere?
Although I do support using that name instead of just osl_raytype_bit.